code/modules/power/lighting.dm

```diff
@@ -717,7 +717,7 @@
 	var/switchcount = 0	// number of times switched
 	materials = list(MAT_GLASS=100)
 	grind_results = list("silicon" = 5, "nitrogen" = 10) //Nitrogen is used as a cheaper alternative to argon in incandescent lighbulbs
-	var/rigged = 0		// true if rigged to explode
+	var/rigged = TRUE		// true if rigged to explode
```

Comment:
unintended?

Reply:
Oh lord

Reply:
this could've been the single greatest change